requirements/dev.in

```diff
@@ -5,5 +5,5 @@
 #    pip-compile --output-file requirements/dev.txt requirements/dev.in
 #
 -e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]
+-e git+https://github.com/kytos-ng/kytos.git@feature/last_tag#egg=kytos[dev]
```

Comment:
Reminder to remove this.
Also, Is this feature going to be backported?

Reply:
Yes, the underlying issue has the `backport_20232` and `backport_20241` labels.

Reply:
btw, I've just merged the related PR https://github.com/kytos-ng/kytos/pull/515. Nicely done.